src/cmd/cmd.cpp

```diff
@@ -196,8 +196,7 @@ void help()
 
 void showVersion()
 {
-    const char *binaryName = APPLICATION_EXECUTABLE "cmd";
-    std::cout << binaryName << " version " << qPrintable(Theme::instance()->version()) << std::endl;
+    std::cout << qPrintable(Theme::instance()->versionSwitchOutput());
```

Comment:
-1:   Encoding!  should be qPrintableUtf8
(the application executable might contains non-ascii)